pkg/operator/ceph/cluster/osd/labels.go

```diff
@@ -32,6 +32,8 @@ const (
 	CephDeviceSetPVCIDLabelKey = "ceph.rook.io/DeviceSetPVCId"
 	// OSDOverPVCLabelKey is the Rook PVC label key
 	OSDOverPVCLabelKey = "ceph.rook.io/pvc"
+	// OSDDeviceClassSetLabelKey is the storageClassDeviceSet label key
+	OSDDeviceClassSetLabelKey = "storageClassDeviceSet"
```

Comment:
How about if we use the existing `CephDeviceSetLabelKey`? The label is already being set on other resources, we just need to add it to the osd job/deployments.

Reply:
Done.